src/cromshell/slim_metadata/command.py

```diff
@@ -11,74 +10,62 @@
 @click.command(name="slim-metadata")
 @click.argument("workflow_id")
 @click.option(
-    "--key",
     "-k",
-    multiple=True,
-    help="Use keys to get a subset of the metadata for a workflow. Multiple keys "
-    "can be set by add option name before adding key (e.g. '-k id -k status ...')",
+    "--keys",
```

Comment:
I'm torn on the multiple use vs comma separated list.  Either works for me.  GATK use multiples but most things use comma separated lists so it's a tossup.  This is fine how it is.